demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -350,18 +374,22 @@ export class DemoMeetingApp {
       );
     }
     this.deviceController = new DefaultDeviceController(logger);
```

Comment:
how does this cloudWatch logger works without meetingId and attendeeId? We cannot create logstream without that right?.
Could we just do console logger instead of multi logger if there is no meeting configuration?